src/main/resources/lessons/webgoat_introduction/documentation/Introduction.adoc

```diff
@@ -1,6 +1,12 @@
 == What is WebGoat?
 ---
 
+ifeval::["{lang}" == "de"]
+:caution-caption: Achtung
+test
```

Comment:
Think this is for test?

Reply:
Yes in the new commit I have support for using the lang as an attribute for use in adoc files, but I don’t think it makes things easier